include/swift/ClangImporter/ClangImporter.h

```diff
@@ -505,10 +512,14 @@ class ClangImporter final : public ClangModuleLoader {
   /// about new Clang modules discovered along the way.
   ///
   /// \returns \c true if an error occurred, \c false otherwise
-  bool addHeaderDependencies(
+  bool getHeaderDependencies(
```

Comment:
This got refactored to instead return a result that the client can use to update the dependency scanner cache, instead of doing it in-place.